frontend/src/components/TileContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import { MouseEvent, ReactElement, useState } from 'react';
+
+import { ACTIVE_TILE_SOURCE, IDLE_TILE_SOURCE } from '../constants/layoutElements';
+import TileState from '../constants/tileState';
+import Tile from '../model/Tile';
+
+interface TileInterface {
+  tile: Tile;
+}
+
+const TileContainer = (props: TileInterface): ReactElement => {
+  const { tile } = props;
+
+  const [currentTileState, setTileState] = useState<TileState>(TileState.ACTIVE);
+
+  function handleActiveTileClick(event: MouseEvent<HTMLImageElement>): void {
+    event.preventDefault();
+    setTileState(TileState.TAKEN);
+  }
```

Comment:
One space before return is a good practice (does not apply to functions that only return something)